daily-challenges/2024-09/19/hunghoang3110.spec.ts

```
@@ -0,0 +1,42 @@
+// Javascript
+let productList = {}
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'productList' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)